opennms-container/horizon/container-fs/confd/templates/slack.properties.tpl

```diff
@@ -0,0 +1,10 @@
+#
+# DON'T EDIT THIS FILE :: GENERATED WITH CONFD
+#
+
+# Configure storage strategy
+org.opennms.netmgt.notifd.slack.channel={{getv "opennms/notifd/slack/channel" "Webhook"}}
```

Comment:
We should add the base path to a variable and then reference it throughout:
```
{{$slackPath := "/opennms/notifd/slack/" -}}

org.opennms.netmgt.notifd.slack.username={{getv (print $slackPath "userName") "none"}}
```

The `print` above concatenates the variable and string to form the full path.

Reply:
Cool. I didn't know that. Thanks!